mne_bids/inspect.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Button
+
+from mne_bids import read_raw_bids, mark_bad_channels
+from mne_bids.read import _from_tsv
+
+matplotlib.use('Qt5Agg')  # XXX Discuss!
```

Comment:
Did you try this out on a mac? JW.

I find some of the qt apps were always buggy on my mac, but that might've changed with qt5.

Reply:
> Did you try this out on a mac? JW.

Yes, I'm using a Mac. `Qt5Agg` is the commended backend for MNE.

What's "JW"?

Reply:
Oh okay awesome! 

Oh.. JW=just wondering. Sorry I abbreviate a lot heh. :p